package.json

```diff
@@ -37,21 +35,36 @@
     "checkbundlesize": "npm run build:prod:standalone:minified && bundlesize",
     "mockserver": "mockserver -serverPort 1080 -jvmOptions \"-Dmockserver.enableCORSForAllResponses=true\""
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "echo 'Running pre-commit scripts...' && lint-staged",
+      "pre-push": "echo 'Running pre-push scripts...' && npm run test"
+    }
+  },
+  "lint-staged": {
+    "./*.js": [
+      "eslint --fix",
+      "git add"
+    ],
+    "./{src,test}/**/*.js": [
+      "eslint --fix",
+      "git add"
+    ],
+    "./*.{html,js}": [
+      "prettier --write",
+      "git add"
+    ],
+    "./{sandbox,src,test}/**/*.{html,js}": [
+      "prettier --write",
+      "git add"
+    ]
+  },
```

Comment:
Yeah, it kind of sucks that these blob matching patterns are duplicated in the `scripts` block and here, but thems the breaks. It would also be nice to be able to combine multiple glob patterns for a particular set of tasks, but it [hasn't been implemented in lint-staged yet](https://github.com/okonet/lint-staged/issues/201).

Reply:
Apparently `./` is important in these glob patterns here because, without it, the file `/Users/aahardy/dev/alloy/test/functional/sandbox/html/linkPageWithoutClickHandler.html` matches both `*.{html,js}` and `{sandbox,src,test}/**/*.{html,js}` and prettier would run twice on the (I found this out by enabling debugging on lint-staged).

However....for the `format` npm script (line 9), we _shouldn't_ use `./` because for whatever reason that forces prettier to stop ignoring `node_modules` so it tries to format `node_modules` files as well (I also found this out by enabling debugging because the process was taking a long time).

I admit I don't really understand the details of why these things are the way they are or if I should be doing something different. It may be intentional by the libraries or simply bugs.